_alerts/2017-09-28-segwit2x-safety.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "Beware of Coinbase, Xapo, and Bitpay!"
+shorturl: "segwit2x-safety"
+active: true
+banner: "Beware of Coinbase, Xapo, and Bitpay!"
+bannerclass: "alert"
+date: 2017-09-28
+---
+
+## Summary
+
+A minority of Bitcoin businesses plan on switching their users to a new incompatible altcoin
+which they plan on wrongly naming Bitcoin through a controversial and contentious hard fork.
+We believe that this new altcoin has no technical legitimacy and will only exist to make it
+easier for corporations to influence the direction of the Bitcoin project. We urge all
+Bitcoin users to immediately remove all their coins from Xapo, Bitpay and Coinbase in particular before November.
+The altcoin these businesses plan on calling Bitcoin will never be Bitcoin. To be safe,
+we recommend that you use Bitcoin Core as your wallet and that you avoid SPV wallets.
+
+Storing any coins on these services is extremely dangerous as these companies openly plan on
+departing from the existing consensus on what Bitcoin actually is. Your actual BTC might be
+frozen and you may have to wait months to obtain them, or they may be lost in replay attacks.
+By behaving in this way, companies like Coinbase, Xapo and Bitpay have shown themselves to be
+enemies of the Bitcoin users. They believe that because you are their customer they have some
+god given right to decide for you the properties of your own money.
+
+Bitcoin is a peer to peer network, the design of the network ensures that some corporate
+elites can't make backroom deals and try to change the network out from under you, but this
+is only true if you run full node software. Depositing coins in services like Coinbase allows
+them to abuse their position and decide for you what is and isn't a Bitcoin, and we believe
+this abuse is likely to cause significant economic damage to you and the wider Bitcoin economy.
+The best course of action is to cease using these services, and to instead migrate all your
+coins to a full node wallet such as Bitcoin Core, as this is the only way to ensure you are
+sovereign over your own money during the controversial hard fork.
+
+## After the fork
+
+As the very first Bitcoin site, and original home of the Bitcoin project launched by Satoshi Nakamoto, we will
+never support Segwit2x, and will remove any wallets or services that present Segwit2x as Bitcoin and not as an altcoin.
+Segwit2x is an attack on the Bitcoin originally envisioned by our site's founder, it essentially plans to turn Bitcoin
+into something similar to PayPal and sets a precedent that rather than the users making their own choices, choices are made
```

Comment:
I disagree with comparing segwit2x to Paypal. I think we should just leave this at "companies deciding what bitcoin is rather than the users".